terraform/default/main.tf

```diff
@@ -1,75 +1,73 @@
 terraform {
   backend "gcs" {
-    bucket  = "constellation-tf"
-    prefix  = "terraform/state-default"
+    bucket = "constellation-tf"
+    prefix = "terraform/state-default"
   }
 }
 
 variable "ssh_user" {
-  type = "string"
+  type = string
 }
 
 variable "project_name" {
-  type    = "string",
+  type    = string
   default = "esoteric-helix-197319"
 }
 
 variable "region" {
-  type    = "string",
+  type    = string
   default = "us-west2"
 }
 
 variable "zone" {
-  type    = "string",
+  type    = string
   default = "us-west2-b"
 }
 
 // Configure the Google Cloud provider
 provider "google" {
- project     = "${var.project_name}"
- region      = "${var.region}"
- version     = "~> 2.0"
+  project = var.project_name
+  region  = var.region
+  version = "~> 2.0"
 }
 
 // Terraform plugin for creating random ids
 resource "random_id" "instance_id" {
- byte_length = 4
+  byte_length = 4
 }
 
 output "cluster_tag" {
   value = "constellation-${random_id.instance_id.hex}"
 }
 
 output "instance_ips" {
-  value = "${module.nodes.instance_ips}"
+  value = module.nodes.instance_ips
 }
 
 output "grafana_ip" {
-  value = "${module.grafana.grafana_ip}"
+  value = module.grafana.grafana_ip
 }
 
 module "network" {
-  source = "./modules/network"
-  random_id = "${random_id.instance_id.hex}"
+  source    = "./modules/network"
+  random_id = random_id.instance_id.hex
 }
 
 module "nodes" {
-  source = "./modules/instance"
-  zone = "${var.zone}"
+  source         = "./modules/instance"
+  zone           = var.zone
   instance_count = 3
```

Comment:
I know it's not for that PR but we should somehow change it dynamically as long as we're specifying number of nodes in deploy scripts

Reply:
we could make it a variable pretty easily